client/checkout/woopay/direct-checkout/woopay-direct-checkout.js

```diff
@@ -173,22 +172,18 @@ class WooPayDirectCheckout {
 			throw new Error( 'Invalid encrypted session data.' );
 		}
 
-		const testMode = getConfig( 'testMode' );
```

Comment:
I'd argue that the `test_mode` was included (https://github.com/Automattic/woopay/pull/2289) in those logs only because they were being passed, and not that we are passing it for logging purposes.

If we need to know the test mode later, we can get it from [the full session](https://github.com/Automattic/woopay/blob/4e031439482d21212fdbd0d42dea0333b851027d/src/Session/RedirectSession.php#L133).

So here I'd stick to only what's important in the request, and log accordingly. LMK if you disagree or have a strong opinion on this.

Reply:
> If we need to know the test mode later, we can get it from [the full session](https://github.com/Automattic/woopay/blob/4e031439482d21212fdbd0d42dea0333b851027d/src/Session/RedirectSession.php#L133).

This is exactly why I thought we may want to include it in the logs. When parsing through the logs, a test mode flag warrants a different response than a production mode log. However, I agree that since the test mode flag is not currently being used, it would be redundant to include it.

> So here I'd stick to only what's important in the request, and log accordingly. LMK if you disagree or have a strong opinion on this.

No strong opinions. I think what you said makes sense 👍